src/posit/connect/permissions.py

```diff
@@ -137,3 +148,86 @@ def get(self, uid: str) -> Permission:
         url = self.params.url + path
         response = self.params.session.get(url)
         return Permission(self.params, **response.json())
+
+    def destroy(self, *permissions: str | Group | User | Permission) -> list[Permission]:
+        """Destroy supplied content item permissions.
+
+        Removes all provided permissions from the content item's permissions.
+
+        Parameters
+        ----------
+        *permissions : str | Group | User | Permission
+            The content item permissions to remove. If a `str` is received, it is compared against
+            the Permissions' `principal_gruid`. If a `Group`, `User`, or `Permission` is received,
+            the `guid` is used and compared against the Permissions' `principal_guid`. If a
+            `Permission` is received, the `principal_guid` is used. Note, only the associated permissions will be destroyed; Any users or groups provided will remain.
```

Comment:
I don't think I follow what this is trying to say? Looking at this + RTFS lower: I think that `Permission` might be wrong in the line above. And then it's saying that if you have a `Group` or a `User`, then the `guid` field from that object will be the `guid` that is used to remove the permission. If you have a `Permission` then the `permission_guid` field is what is used to remove. Is that right?

I wonder if that's a level of detail that is helpful for users or if it's just distracting?

Reply:
Simplified above